### PR TITLE
coconut, assets

### PR DESCRIPTION
**Pull request by nanaian**
🚀

### CODE REVIEW

File: src/code_F5750.c [hunk 1]
```diff
@@ -404,9 +464,75 @@ INCLUDE_ASM(s32, "code_F5750", func_802D3398);
...
+}
+
+ApiStatus func_802D354C(ScriptInstance* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    s32 outVar = *args++;
+    s32* unkStructPtr = get_variable(script, *args++);
+    UnkF5750* temp;
+    s32 i;
+
+    for (i = 0; i < 0x40; i++) {
```
Comment (pixel-stuck) [inline src/code_F5750.c:509]: ARRAY_COUNT here? And below?

File: src/code_F5750.c [hunk 2]
```diff
@@ -404,9 +464,75 @@ INCLUDE_ASM(s32, "code_F5750", func_802D3398);
...
 
-INCLUDE_ASM(s32, "code_F5750", func_802D3474);
+ApiStatus func_802D3474(ScriptInstance* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    s32 outVar = *args++;
+    s32* unkStructPtr = get_variable(script, *args++);
+    UnkF5750* temp;
+    s32 i;
+
+    for (i = 0; i < 0x40; i++) {
```
Comment (pixel-stuck) [inline src/code_F5750.c:474]: similar here and below

File: tools/splat.yaml [hunk 3]
```diff
@@ -9,6 +9,7 @@ options:
   extensions: splat_ext
   symbol_addrs_path: tools/symbol_addrs.txt
   platform: n64
+  assets_dir: assets
```
Comment (pixel-stuck) [inline tools/splat.yaml:12]: I thought assets was the default dir?

File: tools/splat.yaml [hunk 4]
```diff
@@ -3589,9 +3594,9 @@ segments:
     files:
     - [0x929270, c]
     - [0x9292B0, c]
-    - [0x9292D0, .data, world/area_sbk/sbk_00/929270]
-    - [0x929A00, .data, world/area_sbk/sbk_00/9292B0]
-    - [0x929A80, .rodata, world/area_sbk/sbk_00/929270]
+    - [0x9292D0, .data, "929270"]
```
Comment (pixel-stuck) [inline tools/splat.yaml:3597]: ??

**Review by pixel-stuck — CHANGES_REQUESTED** [review]

File: tools/splat.yaml (hunk 4 above)
Comment (nanaian) [reply tools/splat.yaml:3597]: https://hitchdev.com/strictyaml/why/implicit-typing-removed/

File: tools/splat.yaml (hunk 3 above)
Comment (nanaian) [reply tools/splat.yaml:12]: The default behaviour is to not do assets_dir stuff at all, i.e. splat is backwards-compatible

File: src/code_F5750.c (hunk 1 above)
Comment (nanaian) [reply src/code_F5750.c:509]: yes

Comment (nanaian) [reply src/code_F5750.c:509]: @ethteck what the hell man

**Review by pixel-stuck — APPROVED** [review]

**Pull request merged**